config.default

```diff
@@ -2,11 +2,12 @@
   "model": {
     "supported_repos": {
       "repo_kind": [
-        "live",
```

Comment:
So we unable to push a package to, say, live/2.8? It worth to synchronize our plans with Kirill and QA Team.

I would prefer to keep live repositories updated where we already started to do so. It is better to keep it fresh rather than in some unclear state. The new policy that discards live repositories is applied since 2.10.

Reply:
Updated.

Reply:
So how 'enabled' will work? I want to achieve the following (look at the arrows from 'module'):

![image](https://user-images.githubusercontent.com/92749/146375923-6b091d18-8e70-4a1d-98e8-2b82e2cde891.png)

Reply:
A single entry point for pushing a module package to the configurable combination of `repo_kind` with `tarantool_series` will be implemented in https://github.com/tarantool/rws/issues/48
If I understood the question correctly.

Reply:
Perhaps I initially did not understand correctly the task, I can implement #48 in this PR. Up to you.